FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Basic_TLS/mqtt_broker_setup.txt

```diff
@@ -0,0 +1,57 @@
+It is our recommendation to always use strong mutual authentication in any Internet of Things
+application. Instructions below are for setting up a local Mosquitto broker that supports
+TLS server-only authentication for use with this MQTT demo.
+1. Generate certificates with OpenSSL.
+    a. Download and install [Git For Windows](https://git-scm.com/download/win).
```

Comment:
why git is a dependency for mosquitto set up?

Reply:
I'm guessing most users already have it since they had to get it to clone this repo. Also, the [official links to download openssl.exe](https://wiki.openssl.org/index.php/Binaries) do not look reliable.